app/views/support_interface/performance/index.html.erb

```diff
@@ -60,7 +60,7 @@
 </p>
 
 <p class='govuk-body'>
-  <%= govuk_link_to 'Download candidate survey results (CSV)', support_interface_candidate_survey_path %>
+  <%= govuk_link_to 'Download candidate survey results (CSV)', support_interface_candidate_survey_path, download: 'apply-candidate-survey-results.csv' %>
```

Comment:
If there are no survey results the else branch of `SupportInterface#candidate_survey` runs which redirects to a HTML 404 page and we get HTML in the CSV. Should we `head 404, content_type: 'text/csv'` or sth?

![Screenshot 2020-04-27 at 09 25 15](https://user-images.githubusercontent.com/642279/80350699-07154700-8869-11ea-8ad6-758540bfa24a.png)

Reply:
Haha, Quicklook still trying to display that malformed CSV. I'll give that suggestion a go 👍

Reply:
Removed it in the end on the links that don't pertain to this PR!

Reply:
Ah nevermind still need to fix the 404 case... sorry